parlalize/settings/defaults.py

```diff
@@ -135,10 +135,13 @@
 # CORS config
 CORS_ORIGIN_ALLOW_ALL = True
 
+# Legislation status options: in procedure / procedure ended
 LEGISLATION_STATUS = [('v obravnavi', 'v obravnavi'), ('konec obravnave', 'konec obravnave')]
 
+# Legislation result: empty / accepted / rejected
```

Comment:
dodaj:
`# we use these in the django admin interface to define select dropdown options`
`# it is tied to the model, so if you change this please check parlaseje.models.Legislation`